core/src/main/scala/spinal/core/internals/ComponentEmitterVerilog.scala

```diff
@@ -238,10 +238,22 @@ class ComponentEmitterVerilog(
         }
       }
 
+      //Fixing the spacing
+      var maxNameLength = 0
+      for(data <- child.getOrdredNodeIo) {
+        if (emitReferenceNoOverrides(data).toString.length() > maxNameLength) maxNameLength = emitReferenceNoOverrides(data).toString.length()
```

Comment:
It is recommended to use FP style instead of mutable `var` for short and Stable.
``
val maxNameLength = child.getOrdredNodeIo.map(data => emitReferenceNoOverrides(data).toString.length()).max
``